Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1953,19 +2075,512 @@ rideTrackingView push state =
+  , cornerRadius 20.0
+  ][linearLayout 
+    [ height $ WRAP_CONTENT
+    , width $ MATCH_PARENT
+    , clickable true
+    , accessibility DISABLE
+    ][ messagePromtView push state
+     , if ((not $ didDriverMessage FunctionCall) && (state.data.lastSentMessage.sentBy /= "")) then messageView push state.data.lastSentMessage 
+       else if (state.data.lastReceivedMessage.sentBy /= "") then messageView push state.data.lastReceivedMessage 
+       else if os == "IOS" then emptyTextView state else linearLayout[][]
+     , linearLayout
+       [ height $ WRAP_CONTENT
+       , width $ MATCH_PARENT
+       , gravity RIGHT
+       , clickable true
+       , accessibility DISABLE
+       ][ linearLayout
+         [ height $ WRAP_CONTENT
+         , width $ WRAP_CONTENT
+         , cornerRadius 20.0
+         , clickable true
+         , id $ getNewIDWithTag "CrossView"
+         , accessibility ENABLE
+         , accessibilityHint $ "Close : Button : Select to close chat widget"
+         , background Color.manatee200
+         , padding $ Padding 10 10 10 10
+         , onClick push $ const $ RemoveNotification
+         ][imageView
+           [ height $ V 16
+           , width $ V 16
+           , accessibility DISABLE
+           , imageWithFallback $ fetchImage FF_ASSET "ny_ic_cross_white"
+           ]
+         ]
+       ]  
+    ]
+    , separatorView push state
+    , if (state.data.lastMessage.sentBy == "Driver" || not (didDriverMessage FunctionCall)) then quickRepliesView push state else if os == "IOS" then emptyTextView state else linearLayout[][]
+    , if (state.data.lastSentMessage.sentBy /= "" && state.data.lastReceivedMessage.sentBy /= "") then messageView push state.data.lastSentMessage else if os == "IOS" then emptyTextView state else linearLayout[][]
+  ]
+
+separatorView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM ( Effect Unit) w
+separatorView push state = 
+  linearLayout
+  [ height WRAP_CONTENT
+  , width  MATCH_PARENT
+  , margin $ MarginVertical 8 8 
+  ](map (\_ -> linearLayout
+  [ height $ V 1
+  , width $ V 8
+  , margin $ MarginRight 4
+  , background Color.manatee200
+  ][]) (getArray 100))
+
+messageView :: forall w. (Action -> Effect Unit) -> MessagingView.ChatComponent -> PrestoDOM ( Effect Unit) w
+messageView push message =
+  let value = getMessageFromKey message.message $ getValueToLocalStore LANGUAGE_KEY
+  in
+  linearLayout
+  [ width $ V (screenWidth unit - 140)
+  , height $ WRAP_CONTENT
+  , clickable true
+  , onClick push $ const $ MessageDriver
+  , accessibility ENABLE
+  , accessibilityHint $ (if message.sentBy == "Customer" then "You Sent : " else "Message From Driver : ") <> getMessageFromKey message.message "EN_US"
+  ][ imageView
+     [ height $ V 32
+     , width $ V 32
+     , imageWithFallback $ if message.sentBy == "Driver" then fetchImage FF_ASSET "ny_ic_driver_message" else fetchImage FF_ASSET "ny_ic_customer_message"
```

Comment:
use a boolean for this instead of String.

Reply:
didn't have a boolean in the message model